Specs/Data/Cesium3DTiles/Metadata/ImplicitMultipleContentsWithMetadata/tileset_1.0.json

```diff
@@ -0,0 +1,80 @@
+{
+  "asset": {
+    "version": "1.0"
+  },
+  "extensionsUsed": [
+    "3DTILES_implicit_tiling",
+    "3DTILES_multiple_contents",
+    "3DTILES_metadata"
+  ],
+  "extensionsRequired": [
+    "3DTILES_implicit_tiling"
+  ],
+  "extras": {
+    "name": "Sample Implicit Tileset"
+  },
+  "extensions": {
+    "3DTILES_metadata": {
+      "schema": {
+        "classes": {
+          "building": {
+            "properties": {
+              "height": {
+                "type": "SCALAR",
+                "componentType": "UINT16",
+                "semantic": "_BUILDING_HEIGHT"
+              },
+              "color" : {
+                "type": "VEC3",
+                "componentType": "UINT8"
+              }
+            }
+          },
+          "tree": {
+            "properties": {
+              "age": {
+                "type": "SCALAR",
+                "componentType": "UINT16"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "geometricError": 240,
+  "root": {
+    "boundingVolume": {
+      "region": [
+        -1.3197309591796106,
+        0.6988324218,
+        -1.3196290408203893,
+        0.6989155782,
+        0,
+        88
+      ]
+    },
+    "extensions": {
+      "3DTILES_implicit_tiling": {
+        "subdivisionScheme": "QUADTREE",
+        "subtreeLevels": 2,
+        "availableLevels": 2,
+        "subtrees": {
+          "uri": "subtrees/{level}.{x}.{y}.json"
```

Comment:
This points to `0.0.0.json` which is based on the 1.1 schema, even though this tileset uses 3D Tiles 1.0. we should have both `1.0/0.0.0.json` and `1.1/0.0.0.json` like the other implicit examples.

Reply:
I thought content metadata didn't exist for 1.0, so what would a 1.0 subtree look like here?

Reply:
I thought that content metadata wasn't in 1.0, so what would a 1.0 subtree file look like here?

Reply:
Oh I had forgotten that content metadata is new... But then do we need the `tileset_1.0.json` (and any corresponding legacy loading test) for this dataset?

Reply:
Yes, for the implicit tiling extension.